### DISCUSSION TITLE
HTTPS Response Code

### DISCUSSION THREAD

**Discussion by syd20** [cloudprober/cloudprober#222] [Ideas]
Currently AFAIK, the resp_code is returned as a 0 or 1 for each possible http response status. Is there a way to return the actual response code as the value rather than an additional dimension. This would make plotting easier

**Comment by manugarg** [comment]
@syd20 Sorry for noticing this so late. Which plotting software are you using?

**Reply by syd20** [reply]
I use Splunk Observability (APM) to plot the metrics
